blueprint/core/blueprint_ShadowView.cpp

```diff
@@ -150,17 +255,51 @@ namespace blueprint
         template<typename T>
         bool validateFlexProperty (juce::String value, std::map<juce::String, T> validValues)
         {
-            for (const auto& [flexValue, enumValue] : validValues)
+            return std::any_of(validValues.cbegin(), validValues.cend(), [=] (const auto &pair)
```

Comment:
clang-tidy was yelling at me about this. Avoids potential of an unused var in the structured binding and also allows compiler stl loopy unroll shenanigans I guess.

Reply:
:+1: looks good. Nitpicky, but while we're here let's capture `value` by ref and avoid the copy?

Reply:
Yep will update.